### PR TITLE
Add image resources for the demo application

### PR DESCRIPTION
**Pull request by kitchoi**
In #940, I neglected the `images` folder by mistakes. The images in `traitsui/extras/images` are required by the demo application.  This causes the application to show the "?" placeholder image for the "Prev" button etc.

This PR restores these images.

Note that the images are still kept in `traitsui/extras/images` in order to support the legacy script (see #950 for its deprecation).

### CODE REVIEW

**Comment by kitchoi** [comment]
This resolves the following warning from the test suite:
```
test_main (ets-demo.etsdemo.tests.test_main.TestMain) ... /Users/kchoi/.edm/envs/etsdemo-test-3.6-pyqt5/lib/python3.6/site-packages/pkg_resources/__init__.py:1157: DeprecationWarning: Use of .. or absolute path in a resource path is not allowed and will raise exceptions in a future release.
  self, resource_name
```
The fact that the images are not found within `etsdemo` led to pyface searching for various patterns (see https://github.com/enthought/pyface/issues/587)

File: ets-demo/etsdemo/images/image_LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+----------------------------------------------------------------------------
+Nuvola                  LGPL                      image_LICENSE_Nuvola.txt
+
+Unless stated in this file, icons are the work of Enthought, and are
+released under a 3 clause BSD license.
+
+Files and orginal authors:
+----------------------------------------------------------------------------
+examples/tutorials/images:
+                                 next.png | Nuvola
```
Comment (rahulporuri) [inline ets-demo/etsdemo/images/image_LICENSE.txt:14]: nit - alignment is off. not sure why.

File: ets-demo/etsdemo/images/image_LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                         parent.png | Nuvola
+                       previous.png | Nuvola
+                         reload.png | Nuvola
+                            run.png | Nuvola
```
Comment (rahulporuri) [inline ets-demo/etsdemo/images/image_LICENSE.txt:18]: Q : The run.png looks like the default "settings" button :/ not sure if that's what's expected or if it's just mislabeled as `run.png`

**Comment by rahulporuri** [comment]
I have tested this locally and on master, I can reproduce the issue with icons not appearing (or the ? icon being used) and on this branch, the correct icons are being used.

File: ets-demo/etsdemo/images/image_LICENSE.txt (hunk 2 above)
Comment (kitchoi) [reply ets-demo/etsdemo/images/image_LICENSE.txt:18]: I don't know... I just copied the files across without any changes to them.

**Comment by rahulporuri** [comment]
still LGTM.

File: ets-demo/etsdemo/images/image_LICENSE.txt (hunk 2 above)
Comment (rahulporuri) [reply ets-demo/etsdemo/images/image_LICENSE.txt:18]: It looks like `run.png` file is used as an `ImageResource` by `etsdemo.app.DemoFileHandler` but i don't see it in the empty application so i don't know what's happening there.

**Pull request merged**